gui/theme_model.py

```diff
@@ -174,6 +174,29 @@ def value_filter(colorscheme):
 
     {
         'type': 'separator',
+        'display_name': 'Caret'
```

Comment:
and, i think it can be moved above to the "Theme options" since caret is present only in gtk themes